api/apps/api/src/modules/scenarios/scenarios.service.ts

```diff
@@ -13,6 +14,7 @@ import { ScenariosCrudService } from './scenarios-crud.service';
 import { CreateScenarioDTO } from './dto/create.scenario.dto';
 import { UpdateScenarioDTO } from './dto/update.scenario.dto';
 import { UpdateScenarioPlanningUnitLockStatusDto } from './dto/update-scenario-planning-unit-lock-status.dto';
+import { CostSurfaceViewService } from '@marxan-api/modules/scenarios/cost-surface-readmodel/cost-surface-view.service';
```

Comment:
I'd add index.ts 😅

Reply:
changing to `import { CostSurfaceViewService } from './cost-surface-readmodel/cost-surface-view.service';` 🙃

Reply:
fixed in https://github.com/Vizzuality/marxan-cloud/pull/262/commits/d8b24ed044184de9866cc68fdf8e5925ca8cee1d